.github/workflows/CI.yaml

```diff
@@ -78,6 +78,7 @@ jobs:
       - name: Access PLACES_API_KEY
         env:
           MAPS_API_KEY: $ {{ secrets.PLACES_API_KEY }}
+          PLACES_API_KEY: ${{ secrets.PLACES_API_KEY }}
         run: echo PLACES_API_KEY=\"$PLACES_API_KEY\" > ./local.properties
```

Comment:
Did u test this? we should have separate access for MAPS_API_KEY and PLACES_API_KEY? For now I think it is fine since we only use the Places API key

Reply:
Meaning we should write both keys in the local.properties file